demos/link-prediction/attri2vec/stellargraph-attri2vec-DBLP.ipynb

```diff
@@ -30,7 +30,38 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/doc019/.envs/sgdev/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
```

Comment:
could you please clean the notebooks from user-specific metadata?

Reply:
I've cleaned the notebooks and created a new script that auto-cleans the notebooks for us by removing tensorflow warnings, stderr messages, renumbering the cells, and setting the kernelspec to by "Python 3".

The new script is here `scripts/format_notebooks.py`

Reply:
@adocherty nice! Did you write that script yourself, or got it from somewhere? Does it need attributions and CSIRO copyright header?

Reply:
why did you delete the format notebook script?